File: s2/lab2/c++/main.cpp
#include "functions.h"

using namespace std;

int main() {
    create_file("All.txt");
    cout << "List of all workers:" << endl;
    output_file("All.txt");
    create_sorted("All.txt", "Under40.txt", "Over40.txt");
    cout << "List of workers under 40:" << endl;
    output_file("Under40.txt");
    cout << "List of workers over 40:" << endl;
    output_file("Over40.txt");
}

File: s2/lab2/py/main.py
import func


def main():
    func.create_file('All.txt')
    print('List of all workers:')
    func.output_file('All.txt')
    func.get_sorted('Over.txt', 'Under.txt', 'All.txt')
    print('List of workers over 40:')
    func.output_file('Over.txt')
    print('List of workers under 40:')
    func.output_file('Under.txt')


if __name__ == '__main__':
    main()
